apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryJob.cs

```diff
@@ -107,17 +107,19 @@ public BigQueryJob ThrowOnAnyError()
             var errors = Resource.Status?.Errors;
             if (errors?.Count > 0)
             {
-                throw new GoogleApiException(_client.Service.Name, $"Job {Reference?.ProjectId}/{Reference?.Location}/{Reference?.JobId} contained errors")
+                throw new GoogleApiException(_client.Service.Name)
                 {
                     Error = new RequestError
                     {
                         Errors = errors.Select(error => new SingleError
-                            {
-                                Location = error.Location,
-                                Reason = error.Reason,
-                                Message = error.Message
-                            })
-                            .ToList()
+                        {
+                            Location = error.Location,
+                            Reason = error.Reason,
+                            Message = error.Message
+                        }).ToList(),
+                        Message = errors.Count == 1
+                            ? errors.First().Message
+                            : $"Job {Reference?.ProjectId}/{Reference?.Location}/{Reference?.JobId} contained errors"
```

Comment:
Would it be worth including the number of errors here?

Reply:
Yep, I'll do that.